mint-auditor/src/db/models/block_balance.rs

```diff
@@ -70,10 +83,7 @@ impl BlockBalanceModel for BlockBalance {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::db::{
-        block_audit_data::{BlockAuditData, BlockAuditDataModel},
-        test_utils::TestDbContext,
-    };
+    use crate::db::{models::BlockAuditData, test_utils::TestDbContext};
```

Comment:
Nit: Unnecessary import of `BlockAuditData`
```suggestion
    use crate::db::test_utils::TestDbContext;
```